x-pack/test/api_integration/apis/lens/telemetry.ts

```diff
@@ -95,9 +95,10 @@ export default ({ getService }: FtrProviderContext) => {
       const olderDate = moment()
         .subtract(100, 'days')
         .valueOf();
+
+      // @ts-ignore optional type: string
```

Comment:
fix upstream

@joshdover @delvedor do you know the best approach to this?  does the DefinitelyTyped repo have any correlation with es versioning?  does the new client have types we can use?

Reply:
Hello! As far as I know, the type definitions present on DefinitelyTyped are not maintained by us and are out of date (cc @sqren).
The new client offers out of the box the type definitions for every API, but only for url/query parameters, request body and responses are not present because we can't generate them from the spec.
You can see [here](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/typescript.html) how the TypeScript support works :)

Reply:
Just to add to this: 
to work around broken definitely typed definitions in APM we have to overwrite type defs like:

https://github.com/elastic/kibana/blob/2ddbc9eef6cff440dfd264c2cf6f6373280fe5bb/x-pack/legacy/plugins/apm/server/lib/helpers/es_client.ts#L23-L24

Not great but does the job.
Really looking forward to having the new client in Kibana!